extras/parse_travis_log.py

```diff
@@ -392,12 +392,14 @@ def makebuild_summary(log_filename, msg_make_section_start,
 
                 # Only count warnings originating in NEST source files
                 warning_match = nest_warning_re.match(line)
-                if warning_match is not None and line.strip() not in known_warnings:
-                    file_name = line.split(':')[0]
-                    if file_name not in warning_summary:
-                        warning_summary[file_name] = 0
-                    warning_summary[file_name] += 1
-                    number_of_warning_msgs += 1
+                if warning_match is not None:
+                    warning = warning_match.group(0)
```

Comment:
`group(0)` is the full match.
```suggestion
                    warning = warning_match.group(1)
```